packages/web-frontend/src/components/View/utils.js

```diff
@@ -61,6 +61,22 @@ export function getViewWrapper({ type, viewType }) {
   }
 }
 
+export function getExportViewWrapper({ type, viewType }) {
+  switch (type) {
+    case 'chart':
+      return ChartWrapper;
+    default: {
+      return () => (
+        <div>
+          <PDFExportViewTitle>{`'${
+            type || viewType
+          }' type visual does not support PDF export`}</PDFExportViewTitle>
```

Comment:
I'm surprised that there are visuals that don't support PDF export, why is this?

Reply:
Matrix is not exportable, text visuals need a light background to print nicely. Discussion: https://linear.app/bes/issue/RN-494#comment-f866f20a

Reply:
I see. Yep that's all good, can you add that as a comment? We could also add a const like NOT_EXPORTABLE_TYPES to make it explicit, it's hard to understand what type and viewType have to do with anything.